modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -9,81 +9,88 @@ Create a vSphere Windows virtual machine (VM) golden image.
 
 .Prerequisites
 
-* You have installed a cluster on vSphere configured with hybrid networking using OVN-Kubernetes.
-* You have defined a custom VXLAN port in your hybrid networking configuration to work around the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere[pod-to-pod connectivity issue between hosts].
+* You have created a private/public key pair, which is used to configure key-based authentication in the OpenSSH server.
+* You have installed link:https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell[Microsoft PowerShell]. PowerShell commands in this guide are distinguished by the `PS C:\>` prefix.
+
+You have created a private key in the Windows Machine Config Operator (WMCO) namespace. This is required to allow the WMCO to communicate with the Windows virtual machine (VM). See the "Configuring a secret for the Windows Machine Config Operator" section for more details.
```

Comment:
This statement can be combined with the first pre-req "You have created a private/public key pair,..." , since it's referring to the same `private key` from the `private/public key pair`.

Otherwise, may reads that you need a `private/public key pair` and a `private key`